client/src/components/schedule/schedule/Schedule.jsx

```diff
@@ -0,0 +1,175 @@
+import { React, useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import './Schedule.scss';
+import { ButtonSelector } from '../../buttonSelector/buttonSelector/ButtonSelector';
+import { SingleAccordion } from '../../text/Accordion/SingleAccordion/SingleAccordion';
+
+const Schedule = ({ scheduleList }) => {
+  const [activeIndex, setActiveIndex] = useState(0);
+  const days = [];
+  const dayEvents = [];
+  const innerCircleIcons = [];
+  const outerCircleIcons = [];
+  const circleRef = useRef();
+
+  const [x, setX] = useState();
+  const [y, setY] = useState();
+  const getPosition = () => {
+    const x = circleRef.current.offsetLeft;
+    const y = circleRef.current.offsetTop;
+    setX(x);
+    setY(y);
+  };
+
+  useEffect(() => {
+    getPosition();
+  });
+
+  const [windowDimensions, setWindowDimensions] = useState(getWindowDimensions());
+  function getWindowDimensions() {
+    const { innerWidth: width, innerHeight: height } = window;
+    return {
+      width,
+      height,
+    };
+  }
+
+  useEffect(() => {
+    function handleResize() {
+      setWindowDimensions(getWindowDimensions());
+    }
+    window.addEventListener('resize', handleResize);
+    return () => window.removeEventListener('resize', handleResize);
+  }, []);
+
+  for (var i = 0; i < scheduleList.length; i++) {
+    let day = scheduleList[i].date.slice(0, scheduleList[i].date.indexOf(' '));
+    let date = scheduleList[i].date.slice(scheduleList[i].date.indexOf(' '));
+    var buttonDayDate;
+    if (windowDimensions.width > 860) {
+      buttonDayDate = (
+        <div className={'schedule-day-date-desktop'}>
+          <div>{day}</div>
+          <div className={'schedule-date'}>{date}</div>
+        </div>
+      );
+    } else {
```

Comment:
If you want to do this, since different components are loaded
Use the css class
```
.mobile-only {
  display: none;
  @include devices(tablet) {
    display: unset;
  }
}

.desktop-only {
  display: unset;
  @include devices(tablet) {
    display: none;
  }
}
```
in mixins.scss

Reply:
Added